app/js/index.js

```diff
@@ -5,8 +5,10 @@
 var inputOutput;
 var self;
 
+var UI_INSTANCE = new DrawUi();
+
 document.addEventListener('DOMContentLoaded', function() {
-  $('#settingsModal').modal('show');
+  UI_INSTANCE.ShowSettingsDialog();
 }, false);
```

Comment:
Hi @RandomlyKnighted. Thank you for the comment.

Yes. It makes new dependency. But I want to move UI related code to separate file to easily maintain.
Wdyt?